.tool-versions

```diff
@@ -1 +1 @@
-nodejs 22.16.0
+nodejs 22.15.1
```

Comment:
Curious why you downgraded

Reply:
I'm guessing we'd need to run a matrix against a bunch of different node versions to really be confident here